### PR TITLE
Implement ContactClient

### PR DESCRIPTION
**Pull request by grzegorz-jakubiak**
Contact client implementing all actions needed by `meineBIMA` ref: https://github.com/infopark-customers/bima-meinebima2/blob/develop/client/src/stores/contact/store.js

### CODE REVIEW

File: src/clients/contact-client.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  deleteAvatar (id) {
+    return this.client.sendRequest(`${this.client.baseUrl}/${id}/avatar`, {
+      method: 'DELETE'
+    })
+  }
+
+  toggleGravatar (id) {
+    return this.client.sendRequest(`${this.client.baseUrl}/${id}/toggle_gravatar`, {
+      method: 'PUT'
```
Comment (natalka) [inline src/clients/contact-client.js]: Just please double check whenever we have PUT/PATCH here. Probably it's PUT.

File: src/clients/contact-client.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    })
+  }
+}
+
+module.exports = ContactClient
```
Comment (natalka) [inline src/clients/contact-client.js]: hm, does it make standard.js happy?

File: shark-browser.js [hunk 3]
```diff
@@ -22,6 +22,7 @@ Shark.AppClient = require('./src/clients/app-client')
 Shark.AssetClient = require('./src/clients/asset-client')
 Shark.ConsentClient = require('./src/clients/consent-client')
 Shark.ConsentChangesClient = require('./src/clients/consent-changes-client')
+Shark.ContactClient = require('./src/clients/contact-client')
```
Comment (natalka) [inline shark-browser.js]: do we list all clients somewhere in README? If yes, please add it.

File: src/clients/contact-client.js (hunk 1 above)
Comment (grzegorz-jakubiak) [reply src/clients/contact-client.js]: https://github.com/infopark-customers/bima-meinebima2/blob/42ef9a711b6ca4f983332f389ef6a3d936038fbc/client/src/stores/contact/store.js#L149 ;)

File: shark-browser.js (hunk 3 above)
Comment (grzegorz-jakubiak) [reply shark-browser.js]: only in changelog

**Pull request merged**